archivers/xar/Portfile

```diff
@@ -29,6 +29,7 @@ depends_build       port:pkgconfig \
 
 depends_lib         port:bzip2 \
                     port:libxml2 \
+                    path:lib/pkgconfig/icu-uc.pc:icu \
```

Comment:
Doesn't this create a circular dependency with clang-3.7 on older systems?

Reply:
@jmroot since I've made clang-11-bootstrap which builds libcxx it isn't the case anymore.

To be honest: this PR is one of the final steps to make bootstrap on old systems much simpler and faster.

Stay tuned!

Reply:
> @jmroot since I've made clang-11-bootstrap which builds libcxx it isn't the case anymore.
> 
> To be honest: this PR is one of the final steps to make bootstrap on old systems much simpler and faster.

Awesome, great work Kirill!